packages/sfui/frameworks/vue/composables/useScrollable/useScrollable.ts

```diff
@@ -21,14 +21,20 @@ export function useScrollable<TElement extends HTMLElement>(options?: ComputedRe
           state.value = { ...state.value, hasNext: data.hasNext, hasPrev: data.hasPrev };
           options?.value?.onScroll?.(data);
         },
-        onDragChange: (data) => {
+        onDragStart: (data) => {
           state.value = { ...state.value, isDragged: data.isDragged };
-          options?.value?.onDragChange?.(data);
+          options?.value?.onDragStart?.(data);
         },
       });
 
       unregister = scrollable.value?.register();
-      if (options?.value?.activeIndex) scrollable.value.scrollToIndex(options?.value?.activeIndex);
+      if (
+        typeof options?.value?.activeIndex !== 'boolean' &&
+        options?.value?.activeIndex &&
```

Comment:
Probably `activeIndex` could be `0`, and in this case this condition will evaluate to false. Is it expected behaviour?